README.md

```diff
@@ -15,30 +15,97 @@ And you can check all of our open-source projects at [inaka.github.io](http://in
 
 ## Conventions & Rules
```

Comment:
I found 2 misspelled words in this sentence: 
- &
	- *Suggested Replacements:* , k, p, u, e, z, j, &c, o, t, d, y, i, n, s, c, x, h, m, r, b, w, g, l, q, a, v, f
- ##